packages/react-server/package.json

```diff
@@ -28,7 +28,7 @@
     "cookie-parser": "1.3.3",
     "eventemitter3": "0.1.6",
     "express-state": "1.2.0",
-    "lodash": "3.10.1",
+    "lodash": "^4.12.0",
```

Comment:
Nope, just did an `npm install --save lodash@latest`.

I'm not as paranoid about pinning as I once was.  I'd actually like to automatically pick up patch-level changes.

Unless they break everything.  That would suck.